### ISSUE TITLE
Unable to forward to .solutions domain

### ISSUE THREAD

**Issue by Anthracyte** [karan173/Multi-Forward-for-Gmail#3]
Hi,

I like this app and have just started using it and need to forward some mails to my new business email account. It has a .solutions domain address which I think the app doesn't recognise and simply says I must enter a sender and won't forward the mails.  

So if I type the full email address - username@company.solutions the app doesn't forward. If I type something like username@company.solutions.com it then highlights in blue and does allow a send.

**Comment by karan173** [comment]
Yeah this is a bug. The email regex is not flexible enough. Will fix this when I get time.

**Comment by Anthracyte** [comment]
Thanks for the response, appreciate the hard work.
